client/rest-high-level/src/test/java/org/elasticsearch/client/MachineLearningIT.java

```diff
@@ -876,6 +877,16 @@ public void testFilterJob() throws Exception {
         MlFilter createdFilter = putFilterResponse.getResponse();
 
         assertThat(createdFilter, equalTo(mlFilter));
+
+        DeleteFilterRequest deleteFilterRequest = new DeleteFilterRequest(filterId);
```

Comment:
please move this to its own test, almost all other of the CRUD operations are separated out to their own tests.